Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM php:8.1-apache
+
+
+# prep folders and perms
+RUN mkdir -p /var/www/html/episodes
+
+# prep env vars
+ENV \
+    ITEM_COUNT=10000 \
+    MP3_DIR=/var/www/html/episodes \
+    TITLE= \
+    AUTO_SAVE_COVER_ART=
+
+# add any dir2cast.ini options as env vars here
+# todo - making this reusable for all env vars
+RUN echo '; review options in dir2cast.ini source' > /var/www/html/dir2cast.ini
+RUN echo "ITEM_COUNT = ${ITEM_COUNT}" >> /var/www/html/dir2cast.ini
+RUN echo "MP3_DIR = ${MP3_DIR}" >> /var/www/html/dir2cast.ini
+RUN if [[ $TITLE ]]; then echo "TITLE = ${TITLE}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $MP3_URL ]]; then echo "MP3_URL = ${MP3_URL}" >> /var/www/html/dir2cast.ini; fi;
+
+RUN if [[ $IMAGE ]]; then echo "IMAGE = ${IMAGE}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $ITUNES_EXPLICIT ]]; then echo "ITUNES_EXPLICIT = ${ITUNES_EXPLICIT}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $DESCRIPTION ]]; then echo "DESCRIPTION = ${DESCRIPTION}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $ITUNES_AUTHOR ]]; then echo "ITUNES_AUTHOR = ${ITUNES_AUTHOR}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $RECURSIVE_DIRECTORY_ITERATOR ]]; then echo "RECURSIVE_DIRECTORY_ITERATOR = ${RECURSIVE_DIRECTORY_ITERATOR}" >> /var/www/html/dir2cast.ini; fi;
+RUN if [[ $AUTO_SAVE_COVER_ART ]]; then echo "AUTO_SAVE_COVER_ART = ${AUTO_SAVE_COVER_ART}" >> /var/www/html/dir2cast.ini; fi;
+
+# copy source files to docker 
+COPY ./dir2cast.php /var/www/html/
+# skip creating ini. generated above
+# COPY ./dir2cast.ini /var/www/html/
+COPY ./getID3/ /var/www/html/getID3/
```

Comment:
so, the code for getID3 and also the dir2cast.ini file will be downloadable, unless you add some .htaccess to block them. There's instructions for that in the README.

Reply:
This type of server isn't in my expertise so I might be missing something, but it looks like the ini file has to be in the same relative position to the php file the way things are now so it seems like admins would need to set up .htaccess to block whether its running from my docker image or any of the ways listed in the readme

Reply:
Updated with .htaccess to block any .php or .ini requests, and seeing a 403 when I try /getID3/ as the path.